orderer/common/cluster/util_test.go

```diff
@@ -1474,3 +1476,61 @@ func sampleConfigBlock() *common.Block {
 		},
 	}
 }
+
+func TestGetTLSSessionBinding(t *testing.T) {
+	serverCert, err := ca.NewServerCertKeyPair("127.0.0.1")
+	require.NoError(t, err)
+
+	clientCert, err := ca.NewClientCertKeyPair()
+	require.NoError(t, err)
+	srv, err := comm.NewGRPCServer("127.0.0.1:0", comm.ServerConfig{
+		SecOpts: comm.SecureOptions{
+			Certificate:       serverCert.Cert,
+			Key:               serverCert.Key,
+			UseTLS:            true,
+			ClientRootCAs:     [][]byte{ca.CertBytes()},
+			RequireClientCert: true,
+		},
+	})
+	require.NoError(t, err)
+
+	dispatcher := &mocks.Dispatcher{}
+	dispatcher.On("DispatchConsensus", mock.Anything, mock.Anything).Return(nil)
+
+	svc := &cluster.ClusterService{
+		MinimumExpirationWarningInterval: time.Second * 2,
+		StreamCountReporter: &cluster.StreamCountReporter{
+			Metrics: cluster.NewMetrics(&disabled.Provider{}),
+		},
+		Logger:              flogging.MustGetLogger("test"),
+		StepLogger:          flogging.MustGetLogger("test"),
+		Dispatcher:          dispatcher,
+		MembershipByChannel: make(map[string]map[uint64]*msp.SerializedIdentity),
+		AuthVerifier:        &AuthVerifier{},
+	}
+
+	orderer.RegisterClusterNodeServiceServer(srv.Server(), svc)
+	go srv.Start()
+	defer srv.Stop()
+
+	clientConf := comm.ClientConfig{
+		DialTimeout: time.Second * 3,
+		SecOpts: comm.SecureOptions{
+			ServerRootCAs:     [][]byte{ca.CertBytes()},
+			UseTLS:            true,
+			Key:               clientCert.Key,
+			Certificate:       clientCert.Cert,
+			RequireClientCert: true,
+		},
+	}
+	conn, err := clientConf.Dial(srv.Address())
+	require.NoError(t, err)
+
+	cl := orderer.NewClusterNodeServiceClient(conn)
+	stream, err := cl.Step(context.Background())
+	require.NoError(t, err)
+
+	binding, err := cluster.GetTLSSessionBinding(stream, []byte{1, 2, 3, 4, 5})
+	require.NoError(t, err)
+	require.Len(t, binding, 32)
```

Comment:
I am not sure it's enough to check that it's 32 bytes. Can't we also check that the binding is equal in the server side and client side?

Reply:
This testcase is to verify whether the GetTLSSessionBinding is able to generate binding of 32 bytes.  We can add a new testcase to check binding is equal in both client & server in the commauth_test.go

Reply:
then isn't it better to have a single test? Can you please just modify this test?

Reply:
sure. we can have one test. i will update it